src/App.tsx

```diff
@@ -7,25 +7,28 @@ import TravelSection from './components/TravelSection';
 function App() {
   return (
     <div className={styles.app}>
+      <a href="#main-content" className={styles.skipLink}>
+        본문으로 바로가기
+      </a>
       <Navigation />
-      <div className={styles.header}>
+      <header className={styles.header}>
         <h1 className={`${styles.title} heading-1-text`}>A11Y AIRLINE</h1>
         <p className="body-text">
           A11Y AIRLINE은 고객 여러분의 안전하고 쾌적한 여행을 위해 최선을 다하고 있습니다.
         </p>
-      </div>
-      <div id="main-content" className={styles.main}>
-        <div className={styles.flightBooking}>
+      </header>
+      <main id="main-content" className={styles.main}>
+        <section className={styles.flightBooking}>
           <FlightBooking />
-        </div>
-        <div className={styles.travelSection}>
-          <h1 className={`${styles.travelTitle} heading-2-text`}>지금 떠나기 좋은 여행</h1>
+        </section>
+        <section className={styles.travelSection}>
```

Comment:
```suggestion
        <div className={styles.travelSection}>
```
저는 이 안에 감싸고 있는 `TravelSection`의 컴포넌트 이름이 `section` 태그를 사용할 것 같다고 생각해서 이 부분은 `div` 태그를 사용했던 것 같아요!

그리고 현재 코드에 `TravelSection`에 `section` 태그를 사용하는 것의 이점이 있어서 요거는 `TravelSection` 컴포넌트 코드에 코멘트 작성하도록 하겠습니다~